docs/controls/InAppNotification.md

```diff
@@ -84,6 +84,15 @@ ExampleInAppNotification.Dismiss();
 If you want to fully customize the in-app notification, you will see that the Dismiss button is still visible.
 To hide it, simply set the property to `ShowDismissButton="False"`.
 
+### StackMode
+
+By default, each time you display an in-app notification using the same control, each notification will replace the previous one.
+You can change this behavior with one of these values:
+
+* `Replace` - default mode, replace previous notification
+* `QueueBehind` - store every notifications to show, when you dismiss a notification the remaining ones will be displayed successively
+* `StackAbove` - store every notifications to show, when you show a notification it will be displayed in priority (in the reverse order of `QueueBehind` mode)
+
```

Comment:
Not sure if these names make sense for me.  I thought StackAbove at first was going to show multiple notifications on screen at the same time.

Also, shouldn't the be a parameter on the show function vs. a property of the control?  Shouldn't I be able to decide that I have a really important message and want it to be next, but usually I'll just queue them up in order?

Reply:
Well. I am not sure of the use case but the goal was to mimic the behavior of the Microsoft Edge notifications control. When you have multiple notifications, there is only one visible at a time, and displayed successfully when one is dismissed.

For your 1st question, I think this is not required, a property seems better. For your 2nd question, you can surely change the behavior as you go, from `QueueBehind` to `StackAbove` and go back to normal when you want.